jetty-ee10/jetty-ee10-servlet/src/test/java/org/eclipse/jetty/ee10/servlet/ResponseHeadersTest.java

```diff
@@ -204,9 +201,7 @@ public void testMultilineResponseHeaderValue() throws Exception
         assertThat("Response Code", response.getStatus(), is(200));
         assertThat("Response Header Content-Type", response.get("Content-Type"), is("text/plain;charset=UTF-8"));
 
-        String expected = StringUtil.replace(actualPathInfo, "%0A", " "); // replace OBS fold with space
-        expected = URLDecoder.decode(expected, "utf-8"); // decode the rest
-        expected = expected.trim(); // trim whitespace at start/end
+        String expected = URIUtil.canonicalPath(actualPathInfo);
```

Comment:
Looks like you changed the expectations to make the test pass.
Why?

Reply:
Looking back at the history of this test, this is a test to prevent a regression.

There was a time where a response header value could (accidentally?) trigger a response header formatted with the old "obsolete fold" http syntax.
This test was to ensure that this obsolete fold format was not used.

Reply:
I will fully decode the `pathInfo` in the servlet and then send it back as a header. 
Then we can keep this test expectation the same.